quickstart/src/index.js

```diff
@@ -7,32 +7,50 @@ var previewTracks;
 var identity;
 var roomName;
 
-// Attach the Tracks to the DOM.
-function attachTracks(tracks, container) {
-  tracks.forEach(function(track) {
-    container.appendChild(track.attach());
-  });
+
+// Attach given track to the DOM.
+function attachTrack(container, track) {
```

Comment:
@makarandp0 Nitpick: Since the function is about the Track, I think it should come first.